abixen-platform-web-client/src/main/web/common/modal-selection/html/modal-selection.html

```diff
@@ -18,8 +18,8 @@ <h3 class="modal-title">{{title}}</h3>
       <span ng-if="selectionType == 'MULTIPLE'" class="animate-if">
           <div class="container-fluid pre-scrollable" >
           <div class="row"><!-- pre-scrollable-->
-              <div class="col-sm-4 btn" style="border:0px solid transparent;" ng-repeat="rec in data" ng-click="addToSelection(rec)">
-                  <div class="col-sm-10 col-sm-3" style="padding:10px;cursor:pointer;"><i class="{{rec.text}}" aria-hidden="true"></i>&nbsp;{{rec.text}}</div>
+              <div class="col-sm-6" style="border:0px solid transparent;" ng-repeat="rec in data" ng-click="addToSelection(rec)">
+                  <div class="col-sm-10 col-sm-2" style="padding:10px;cursor:pointer;text-align:center;"><i class="{{rec.text}}" aria-hidden="true"></i>&nbsp;{{rec.text}}</div>
```

Comment:
As discussed, I'll use another task/commit to move all my css to sass file.

Please review and approve.